client/js/widgets/widget.js

```diff
@@ -2067,4 +2092,14 @@ export class Widget extends StateManaged {
   validDropTargets() {
     return getValidDropTargets(this);
   }
+
+  inheritedSeatVisibility() {
```

Comment:
Other widget methods are in alphabetical order.

Reply:
Thanks, moved to the sorted location. `addAudio` looks out of order but I prefer to keep cleanups of unrelated code separate.